.werft/build.yaml

```diff
@@ -53,7 +53,7 @@ pod:
     - name: MYSQL_TCP_PORT
       value: 23306
   - name: build
-    image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:as-migrate-mixins.73
+    image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:pd-update-to-node-14-fork.0
```

Comment:
Please do not introduce this change in one commit. We usually introduce the change and in a different one the update to the images.

You need to search/replace all the references to the old image

Reply:
Yes, that way is easy to track the changes to images